src/data/move.ts

```diff
@@ -9812,7 +9931,7 @@ export function initMoves() {
       .attr(StatStageChangeAttr, [ Stat.ATK ], -1),
     new StatusMove(Moves.INSTRUCT, Type.PSYCHIC, -1, 15, -1, 0, 7)
       .ignoresSubstitute()
-      .unimplemented(),
+      .attr(RepeatMoveAttr),
```

Comment:
```suggestion
      .attr(RepeatMoveAttr)
      .edgeCase(), // incorrect interactions with Gigaton Hammer, Blood Moon, Torment, and the Pledges
```
Adding an edge case for the buggy interactions. I figure the "problem moves" are specific enough to not warrant a `.partial()` marker.